musabase/cgiar_banana_ontology.obo

```diff
@@ -5706,6 +5706,229 @@ namespace: BananaTrait
+relationship: variable_of CO_325:0100008 ! cm
+
+[Term]
+id: CO_325:0003007
+name: Diameter at 10 cm from the stellar region after weevil introduction
+namespace: BananaTrait
+def: "slice the corm measure the diameter using a tape (cm) 35 and 66 days after weevil introduction (experiement done in potted plant) In cm" []
+is_a: CO_325:1000024 ! Variable
+relationship: variable_of CO_325:0003004 ! weevil damage assessment
+relationship: variable_of CO_325:0010262 ! corm size at 10 cm - measurement
+relationship: variable_of CO_325:0100008 ! cm
+
+[Term]
+id: CO_325:0003008
+name: Inner damage percentage at the stellar region after weevil introduction
+namespace: BananaTrait
+is_a: CO_325:1000024 ! Variable
+relationship: variable_of CO_325:0003004 ! weevil damage assessment
+relationship: variable_of CO_325:0010155 ! corm size - measurement
+relationship: variable_of CO_325:0100046 ! %
+
+[Term]
+id: CO_325:0003009
+name: Outer damage percentage at 5 cm from the stellar region after weevil introduction
+namespace: BananaTrait
+is_a: CO_325:1000024 ! Variable
+relationship: variable_of CO_325:0003004 ! weevil damage assessment
+relationship: variable_of CO_325:0010264 ! outer damage at 5 cm - measurement
+relationship: variable_of CO_325:0100046 ! %
+
+[Term]
+id: CO_325:0003010
+name: Outer damage percentage at 10 cm from the stellar region after weevil introduction
+namespace: BananaTrait
+is_a: CO_325:1000024 ! Variable
+relationship: variable_of CO_325:0003004 ! weevil damage assessment
+relationship: variable_of CO_325:0010263 ! outer damage at 10 cm - measurement
+relationship: variable_of CO_325:0100046 ! %
+
+[Term]
+id: CO_325:0003011
+name: Inner damage percentage at 5 cm from the stellar region after weevil introduction
+namespace: BananaTrait
+is_a: CO_325:1000024 ! Variable
+relationship: variable_of CO_325:0003004 ! weevil damage assessment
+relationship: variable_of CO_325:0010266 ! inner damage at 5 cm - measurement
+relationship: variable_of CO_325:0100046 ! %
+
+[Term]
+id: CO_325:0003012
+name: Inner damage percentage at 10 cm from the stellar region after weevil introduction
+namespace: BananaTrait
+is_a: CO_325:1000024 ! Variable
+relationship: variable_of CO_325:0003004 ! weevil damage assessment
+relationship: variable_of CO_325:0010265 ! inner damage at 10 cm - measurement
+relationship: variable_of CO_325:0100046 ! %
+
+[Term]
+id: CO_325:0003014
+name: nematode damage
+namespace: BananaTrait
+is_a: CO_325:1000002 ! Biotic_stress
+
+[Term]
+id: CO_325:0003015
+name: Helicotylincus mulicinctus damage
+namespace: BananaTrait
+def: "Helicotylincus mulicinctus damage assesssment" []
+is_a: CO_325:0003014 ! nematode damage
+
+[Term]
+id: CO_325:0003016
+name: Number of female Helicotylincus mulicinctus - counting
+namespace: BananaTrait
+def: "Measurement per 100 grams of soil sample" []
+is_a: CO_325:1000024 ! Variable
+relationship: variable_of CO_325:0003015 ! Helicotylincus mulicinctus damage
```

Comment:
shouldn't it be linked to 
id: CO_325:0000523
name: number of helicotylenchus multicinctus per soil sample
instead?

Reply:
yes, I'll fix this and the same for the rest of the 'per soil sample' variables